### DISCUSSION TITLE
How to flatten all form fields in PDF files ?

### DISCUSSION THREAD

**Discussion by hoaile-novobi** [pymupdf/PyMuPDF#1928] [Q&A] [answered]
Is there any method to flatten all the form fields of PDF files ?

**Comment by JorjMcKie** [comment, marked as answer]
Use the `Document.convert_to_pdf()` method.
This may **_not_** work for PDFs with font problems like missing ToUnicode mappings, but certainly worth a try.

**Reply by hoaile-novobi** [reply]
Thanks, it worked for me now.
